docs/reference/indices/aliases.asciidoc

```diff
@@ -312,7 +312,12 @@ Possible options:
 
 The rest endpoint is: `/{index}/_alias/{alias}`.
 
+ifdef::asciidoctor[]
+added[1.4.0.Beta1,"The API will always include an `aliases` section, even if there aren't any aliases. Previous versions would not return the `aliases` section"]
```

Comment:
Could you do `added::[` here?

Reply:
Yes. Added with bc378ca96f8f5fa0adad36f1492f907905b50097. Thanks @nik9000 !